creation/interfaces-and-networks.md

```diff
@@ -692,3 +681,117 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
 > into VMI guest. Because of that, when configuring SR-IOV operator
 > policies, make sure you define a pool of VF resources that uses
 > `driver: vfio`.
+
+Once the operator is deployed, an [SriovNetworkNodePolicy
+](https://github.com/openshift/sriov-network-operator#sriovnetworknodeconfigpolicy)
+must be provisioned, in which the list of SR-IOV devices to expose (with
+respective configurations) is defined.
+
+Please refer to the following `SriovNetworkNodePolicy` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 9000
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov-nic
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriovgnic`.
+
+> **Note:** Further instructions will assume the aforementioned operator
```

Comment:
This should come after you mention the operator. At this point, you've already used it for the Policy

Reply:
OK, shouldn't have listened to you in the first place and moved it around, since it was in the right place: before I assume the operator was used. Getting rid of the note altogether.